portfolio/src/main/webapp/script2.js

```diff
@@ -0,0 +1,8 @@
+async function displayFacts(){
+const serverRequest = await fetch("/funfacts");
+const messageFromServer = await serverRequest.text();
+
+const factsContainer = document.getElementById("funFacts");
+factsContainer.innerText = messageFromServer;
+
+}
```

Comment:
To make the code more readable, could you format the code with correct indentation.

Reply:
Yes. I will do that. Thanks for the eye.  I didn't notice the indentation was off